pkg/cmd/kind/create/cluster/createcluster.go

```diff
@@ -51,6 +52,9 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 		Short: "Creates a local Kubernetes cluster",
 		Long:  "Creates a local Kubernetes cluster using Docker container 'nodes'",
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if !cmd.Flags().Changed("name") {
```

Comment:
we could dry this out by just passing the `cmd.Flags()` (flagset pointer) to `OverrideDefaultName` and then internally check `flags.Changed("name")` then call `flags.Set("name", value)`

Reply:
Done.
I initially had that, but figured that checking the specific `name` flag logic should probably live near the command than in the method that just checks the env.
/shrug